api/src/main/java/org/pmiops/workbench/captcha/CaptchaVerificationServiceImpl.java

```diff
@@ -17,7 +21,21 @@
 
   final String urlPattern = "https://%s/login";
 
-  final String localHostUrlPattern = "http://%s:4200/login";
+  final List<String> nonTestCaptchaURLs =
```

Comment:
I would rather drive this from a config value per environment, unless there's something I'm missing. This list approach seems brittle to me.

Reply:
I was not sure if just checking google test key should be enough for test/staging/perf and local. Normally for captcha verification we would check site key + host matches the url (the way we are doing it for prod and staable). But again since i am checking url against the config entry i can just create a new entry to test if the environment is using testCaptcha